cmd/dep/status_test.go

```diff
@@ -26,3 +30,38 @@ func TestStatusFormatVersion(t *testing.T) {
 		}
 	}
 }
+
+func TestBasicLine(t *testing.T) {
+
+	project := dep.Project{}
+
+	var tests = []struct {
+		status   BasicStatus
+		expected string
+	}{
+		{BasicStatus{
+			Version:  nil,
+			Revision: gps.Revision("flooboofoobooo"),
+		}, `[label="\nflooboo"];`},
+		{BasicStatus{
+			Version:  gps.NewVersion("1.0.0"),
+			Revision: gps.Revision("flooboofoobooo"),
+		}, `[label="\n1.0.0"];`},
+	}
+
+	for _, test := range tests {
+		var buf bytes.Buffer
+
+		out := &dotOutput{
+			p: &project,
+			w: &buf,
+		}
+		out.BasicHeader()
+		out.BasicLine(&test.status)
+		out.BasicFooter()
+
+		if ok := strings.Contains(buf.String(), test.expected); !ok {
+			t.Fatalf("Did not find node with identifier string %s in digraph", test.status)
```

Comment:
Can you make it something like `"Did not find expected node label: \n\t(GOT) %v \n\t(WNT) %v"`? That's kind of the convention we follow, got and want.
Overall, it looks good 👍